client/src/modules/journal/journal.js

```diff
@@ -169,13 +169,8 @@ function JournalController(
     headerCellFilter : 'translate',
     visible : false,
     enableCellEdit : false,
-  }, {
-    field : 'period_end',
-    displayName : 'TABLE.COLUMNS.PERIOD',
-    headerCellFilter : 'translate',
-    cellTemplate : 'modules/templates/bhPeriod.tmpl.html',
```

Comment:
Does any module still use this template?  If not, can it be removed?

Reply:
Ah, looks like it is used [here](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/modules/account_statement/account_statement.ctrl.js#L126).